Dockerfile.rhel7

```diff
@@ -1,26 +1,43 @@
-FROM rhel7
+FROM playbook2image
```

Comment:
I'll reference the image as `openshift3/playbook2image` for the time being - maybe still not final, but at least it will have a namespace...